rest/handler.go

```diff
@@ -304,6 +304,30 @@ func (h *handler) checkAuth(context *db.DatabaseContext) (err error) {
 
 	}(time.Now())
 
+	// Is there a TLS client cert?
+	var clientCertEmailAddress *string
+	if clientCerts := h.rq.TLS.VerifiedChains; len(clientCerts) > 0 {
+		clientCert := *clientCerts[0][0]
```

Comment:
AFAIK a client can only send one cert, while a server can send multiple. (Actually by 'cert' here I mean 'cert chain', i.e. the cert plus copies of the ones that signed it.) So I'm a bit confused why the Go API uses `[][]*Cert` here. It may be because `VerifiedChains` is used on both client and server? I'll investigate.

Reply:
Verified that a client can only send one cert.